ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -2138,6 +2138,13 @@ void ActivityInstance::createGraphNode(IPropertyTree * defaultSubGraph, bool alw
             getRecordCountText(text, dataset);
             addAttribute("predictedCount", text);
         }
+
+        if (options.addLikelihoodToGraph && kind==TAKfilter)
```

Comment:
This code should be in the code that builds the filter condition, rather than here.  (See above for some reasons why